.lando.yml

```diff
@@ -1,7 +1,7 @@
 name: ccr
 recipe: laravel
 config:
-  php: "8.0"
+  php: "7.4"
```

Comment:
Just wondering the reasoning behind the downgrade of php

Reply:
There's a bug in lando in PHP 8. I submitted a PR to their repo to fix it but it hasn't been merged yet.

Reply:
Just checked and it looks like it's since been merged and is fixed in Lando 3.0.25. I'll update things back to php 8. Just note that you'll need to upgrade Lando.